shared/fs/footer/upload-container.js

```diff
@@ -33,16 +34,29 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   debugToggleShow: getDebugToggleShow(dispatch),
 })
 
-const mergeProps = ({_uploads}, {debugToggleShow}) =>
-  ({
+const mergeProps = ({_uploads, _pathItems}, {debugToggleShow}) => {
+  const folders = _pathItems.filter(path => path.type === 'folder')
+  const uploadPaths = _uploads.syncingPaths.toJS() // get array of paths
+  const filePaths = []
+
+  uploadPaths.map(path => {
+    if (!folders.has(path)) {
+      //  path is not part of the folders
+      filePaths.push(path)
+    }
+  })
```

Comment:
Several things:

1. I think we'd want the `files` prop to only include files anyway, and `syncingPaths` has both files and folders now. So for example, if you are uploading `/keybase/team/blah/a/b.txt`, `syncingPaths` would include both `/keybase/team/blah/a` and `/keybase/team/blah/a/b.txt`. In this case, ideally we'd want `files` to be 1 instead of 2.

2. I think we should show the file name instead of file path. There's `Constants.getPathName` that can be useful.

3. The component only need to know about file name if it's the only one. So `filePaths` should be a single string `fileName`.

If you take my suggestion above about using `filter` on `syningPaths`, then we'd get a `uploadPaths` that only includes files. So `files` can just be the size of it, and `fileName` would be either null or undefined, or the only thing from `uploadPaths`.